doc/tutorials/klondike/app/lib/step5/klondike_game.dart

```diff
@@ -0,0 +1,299 @@
+import 'dart:math';
+import 'dart:ui';
+
+import 'package:flame/components.dart';
+import 'package:flame/flame.dart';
+import 'package:flame/game.dart';
+
+import 'components/card.dart';
+import 'components/flat_button.dart';
+import 'components/foundation_pile.dart';
+import 'components/stock_pile.dart';
+import 'components/tableau_pile.dart';
+import 'components/waste_pile.dart';
+
+enum Action { newDeal, sameDeal, changeDraw, haveFun }
+
+class KlondikeGame extends FlameGame {
+  static const double cardGap = 175.0;
+  static const double topGap = 500.0;
+  static const double cardWidth = 1000.0;
+  static const double cardHeight = 1400.0;
+  static const double cardRadius = 100.0;
+  static const double cardSpaceWidth = cardWidth + cardGap;
+  static const double cardSpaceHeight = cardHeight + cardGap;
+  static final Vector2 cardSize = Vector2(cardWidth, cardHeight);
+  static final cardRRect = RRect.fromRectAndRadius(
+    const Rect.fromLTWH(0, 0, cardWidth, cardHeight),
+    const Radius.circular(cardRadius),
+  );
+
+  // Constant used to create Random seed. Cannot use 64 bits in Flame Web apps.
+  // Also Random.nextInt(n) throws an exception if n > 32 bits. 52 factorial,
+  // the number of permutations of 52 cards, is of order 10 to the power 68!
+  static const int maxInt = 4294967296; // This is 2 to the power 32.
```

Comment:
According to the docs this should be:
```suggestion
  static const int maxInt = (1 << 32) - 1;
```
The -1 is needed since it is a signed.

Reply:
In a Chrome build, `(1 << 32) - 1` actually evaluates to -1 and throws an exception at runtime. I have written it now as a hex value, with an explanatory comment.